src/encoded/schemas/chia_pet_alignment_quality_metric.json

```diff
@@ -23,39 +23,51 @@
         },
         "total_rp": {
             "title": "Number of total read pairs in raw data",
-            "type":  "integer",
+            "type": "integer",
+            "description": "Total read pairs",
+            "comment": "It is number of read pairs from fastq files",
             "minimum": 0
         },
         "rp_bl": {
             "title": "Number of read pairs with bridge linker",
             "type":  "integer",
+            "description": "Read pairs with linker",
+            "comment": "We used 20bp our bridge-linker sequence, and allow 2bp mismatch to detect a linker sequence",
```

Comment:
I am not 100% sure about this, but is "20bp our" supposed to be "20bp for our"?

Reply:
fixed